### PR TITLE
[Console] [components] add information about the addRows method - table.rst

### PR DESCRIPTION
**Pull request by Sarah-eit**
Added information about the "addRows" method.

File : table.rst

### CODE REVIEW

File: components/console/helpers/table.rst [hunk 1]
```diff
@@ -432,3 +432,24 @@ This will display the following table in the terminal:
...
+
+        // ...
+        $table->addRows([
+            ['Hello', 'World'],
+            ['Love', 'Symfony'],
+        ]);
+        $table->render();
+        // ...
+
+    This generates:
```
Comment (94noni) [inline components/console/helpers/table.rst]:
```suggestion
    This will output:
```
```suggestion
    This will display:
```

wdyt?

**Review by 94noni — APPROVED** [review]
nice addition

File: components/console/helpers/table.rst (hunk 1 above)
Comment (Sarah-eit) [reply components/console/helpers/table.rst]: I will send the correction with your second proposal.
Thank you.

**Comment by javiereguiluz** [comment]
Thank you Sarah.

**Pull request merged**